cc_embed_data/BUILD

```diff
@@ -36,8 +36,8 @@ cc_flags_supplier(
 )
 
 cc_binary(
-    name = "make_emebed_data",
-    srcs = ["make_emebed_data.cc"],
+    name = "make_embed_data",
```

Comment:
Please add a deprecated alias to in place of the old rule.
https://docs.bazel.build/versions/main/be/general.html#alias

Include in the deprecation message a short explanation of this change. ... That is assuming that the deprecation message gets printed before Bazel errors out due to something depending on the renamed files. If it usually fails that way, just say so here and no need to bother.

Reply:
Out of curiosity: from where did you find this library?